### DISCUSSION TITLE
Server started but cannot reach <ip>:7777

### DISCUSSION THREAD

**Discussion by TaihouKai** [Pryaxis/TShock#1698] [Q&A] [answered]
I started TShock service using [this tutorial](https://stackoverflow.com/questions/54709898/how-to-install-terraria-tshock-server-on-ubuntu-server), but I stuck on "connecting to ....".

So I tried `https://www.yougetsignal.com/tools/open-ports/`, and it says...
Port 7777 is closed on 207.148.114.144.

PS: I tried to ping my server ip and it worked properly.

**Comment by TaihouKai** [comment, marked as answer]
OK, I forgot port forwarding...

**Reply by hakusaro** [reply]
This is the best post so far.
